src/bin/travis/resources/papi_conformance_expected_failures.txt

```diff
@@ -114,6 +114,6 @@
 114
 115
 116
-#117
+117
```

Comment:
😢

Reply:
It's not as bad as you think, it's a pre-processing conformance test problem where we don't prefix default values in the CWL with gcs paths.
We'll fix it at some point it's just not in the set of #1 priorities now 😄